sources/grcc.cc

```diff
@@ -55,8 +55,8 @@ using namespace Grcc;
 
 static OptDef optDef[] = {
  {"Step",       "Generate particle assigned graphs",               GRCC_AGraph, 0},
- {"Outgrf",     "Ouput to file (out.grf)",                               False, 0},
- {"Outgrp",     "Ouput to file (out.grp)",                               False, 0},
+ {"Outgrf",     "Output to file (out.grf)",                               False, 0},
+ {"Outgrp",     "Output to file (out.grp)",                               False, 0},
  {"1PI",        "Only 1PI graphs",                                       True , 0},
```

Comment:
Sorry for nitpicking, but could you align `False, 0}` with the other lines?

Reply:
done